src/Sylius/Bundle/WebBundle/Behat/WebContext.php

```diff
@@ -713,6 +725,20 @@ private function assertRoute($route)
         }
     }
 
+    protected function assertCouponsCodeLength(array $coupons, $expectedLength)
+    {
+        $expectedLength = (int) $expectedLength;
```

Comment:
I think we can check length only of the first coupon. Then we can write this check as:

```php
if (strlen(current($coupons)) !== $expectedLength) {
  throw new Exception();
}
```